script/app.js

```diff
@@ -0,0 +1,106 @@
+//1.написать функцию  filterFalsyValues, которая принимает массив и возвращает массив без '', null, NaN, false, undefined, 0:
+function filterFalsyValues(arr) {
+  return arr.filter(function(el) {
+   if (el === '') {return false;}
+   if (el === null) {return false;}
+   if (el === false){return false;}
+   if (el === undefined) {return false;}
+   if (el === 0) {return false;}
+   if( Number.isNaN(el)){return false;}
+   else {return true;}
+  });
+}
+console.log(filterFalsyValues([5, 'asd', 'false', true, false, null, NaN, undefined,0]))//[5, 'asd', 'false', true]
+
+// 2.написать функцию getMultipliedBigNumbers, которая принимает один прараметр  - массив чисел, и умножает  каждое число в массиве  само
+// на себя, и возвращает массив с числами  больше 1000:
+
+function getMultipliedBigNumbers(arr) {
+  let newArr = [];
+     arr.reduce(function(result, item, i){
+       let res = item * arr[i];
+      if (res > 1000 ) {
+        return newArr.push(res)
+      };
+      return result;
+  },arr);
+  return newArr;
+}
+console.log(getMultipliedBigNumbers([7, 12, 20, 50, 70, 100]))//[2500, 4900, 10000]
+
+//3.написать функцию getArraysValuseSum ,которая принимает  массив, и возвращает сумму всех чисел в массиве, если элемент массива
+// не число, а строка, то суммирует  кол-во букв в строке:
+
+function getArraysValuseSum(arr) {
+  let newArr = [];
+  arr.reduce(function(result,item) {
+    if (typeof(item) === 'string') {
+      return newArr.push(item.length);
+    } else if (typeof(item) === 'number') {
+      return newArr.push(item);
+    }
+   },arr);
+   newArr = newArr.reduce(function(result, item) {
```

Comment:
you don't need second reduce.
run first reduce with `initialValue = 0`, not `arr`.
should look like that:
`function getArraysValuseSum(arr) {
  return `arr.reduce(.......);
  }
nothing else